builds/misc/templates/build-rust-artifacts.yaml

```diff
@@ -15,19 +29,19 @@ steps:
     inputs:
       filePath: scripts/linux/cross-platform-rust-build.sh
       arguments: --os ubuntu18.04 --arch aarch64 --build-path edge-hub/watchdog
-  # Build broker
   - task: Bash@3
-    displayName: Build MQTT Broker - amd64
+    displayName: Build generic mqtt tester - amd64
     inputs:
       filePath: scripts/linux/cross-platform-rust-build.sh
-      arguments: --os alpine --arch amd64 --build-path mqtt/mqttd
+      arguments: --os alpine --arch amd64 --build-path test/modules/generic-mqtt-tester
   - task: Bash@3
-    displayName: Build MQTT Broker - arm32
+    displayName: Build generic mqtt tester - arm32
     inputs:
       filePath: scripts/linux/cross-platform-rust-build.sh
-      arguments: --os ubuntu18.04 --arch arm32v7 --build-path mqtt/mqttd
+      arguments: --os ubuntu18.04 --arch arm32v7 --build-path test/modules/generic-mqtt-tester
   - task: Bash@3
-    displayName: Build MQTT Broker - arm64
+    displayName: Build generic mqtt tester - arm64
     inputs:
       filePath: scripts/linux/cross-platform-rust-build.sh
-      arguments: --os ubuntu18.04 --arch aarch64 --build-path mqtt/mqttd
+      arguments: --os ubuntu18.04 --arch aarch64 --build-path test/modules/generic-mqtt-tester
```

Comment:
I would try to avoid duplicating some of this code, but since display name cannot contain dynamic text I think it is better to have duplication here and an understandable description we can see in the build pipeline

Reply:
I see a few different places in our yaml files where we build the displayName from variables...is this what you're talking about?

Examples:
```yaml
displayName: Linux Rust ${{ parameters.archName }}
displayName: Build ($(Build.Configuration))
```

Reply:
For the second one I think this might work because it is available as soon as the build starts, but I wouldn't expect the first to work based on this:
https://github.com/MicrosoftDocs/azure-devops-docs/issues/2327

Also I couldn't find the first example in our repo.

Reply:
IMO it's your call...

The first example is from the `release/1.1` branch, doesn't exist in master.